bokehjs/examples/linked/linked.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>Linked Brushing &amp; Panning</title>
+    <link rel="stylesheet" href="../../build/css/bokeh.css" type="text/css" />
+    <script type="text/javascript" src="../../build/js/bokeh.js"></script>
+  </head>
+  <body>
+    <script type="text/javascript" src="linked.js"></script>
+  </body>
+</html>
```

Comment:
Currently `anscombe` example has additional content to show how to embed without `plt.show()`. There may be new examples needing custom content as well. We will see, maybe some can be auto-generated.